### DISCUSSION TITLE
KMonad command not recognized after installing on MacOS Monterey

### DISCUSSION THREAD

**Discussion by Christensen-John** [kmonad/kmonad#757] [Q&A]
I failed to correctly install Kmonad three times. I'm hoping to run some questions past folks here to have a more successful attempt.

1. Is it necessary to have Karabiner Elements installed or does the section for Dext fill that need?
2. After the Dext section (where the instructions say to clone the Kmonad git repo) the instructions directly below it say to clone the git repo again. Is that necessary? Do I stay in the KMonad directory that I was in during the previous steps?
3. Is it normal to get errors during the installation (messages about deprecation and such)?
4. I tried running the KMonad command in terminal after the last install (which seemed to go more smoothly) but I received the message `Command not found` but I hadn't seemed to have received any install breaking errors. Was I testing it incorrectly?

Thanks for your help!

**Comment by robrecord** [comment]
I've not seen much in the way of support for later macOS versions - perhaps this project needs an upgrade to work on Monterey and beyond.

**Reply by robrecord** [reply]
Actually it has been made to work, the docs have just not been updated yet... still figuring it out though!

**Comment by robrecord** [comment]
2. No you don't need to clone again.

**Comment by slotThe** [comment]
> the docs have just not been updated yet...

Would you care to update them? :)

**Reply by robrecord** [reply]
I'm a complete newcomer to kmonad and just about managed to cobble together an install from the various bits of info, so it would be the blind leading the blind, but if I get some free time I can try.
